docs/intro.rst

```diff
@@ -239,14 +261,38 @@ environment. You can detect that bit size as follows:
   Swig at their installation time. Therefore, there are no prerequisite
   OS-level packages to install.
 
-* Install pywbem (and its prerequisite Python packages) into the active
+* Install pywbem (and its other prerequisite Python packages) into the active
   Python environment:
 
   .. code-block:: bash
 
       > pip install pywbem
 
 
+.. _`Installing to a UNIX-like environment under Windows`:
+
+Installing to a UNIX-like environment under Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section describes the installation of pywbem into a UNIX-like environment
+under Windows (such as CygWin, MinGW, Babun, or Gow).
+
+Note that Unix-like environments on Windows bring their own Python, so in such
+an environment, you install into that Python, and not into the Python of
+Windows.
+
+* Install pywbem (and its other prerequisite Python packages) into the active
+  Python environment:
+
+  .. code-block:: bash
+
+      > pip install pywbem
+
+  If the Swig compilation during installation of M2Crypto fails, there may be
+  components of the UNIX-like environment missing (e.g. OpenSSL development,
+  Python development, Swig, gcc).
```

Comment:
Add the following: At least under cygwin, these may be installed using the cygwin installer (openssl-devel, python2-devel or python3-devel), swig).

Reply:
Two comments:
* python3-devel will never be needed, because M2Crypto is only used on Python 2.
* The gcc package on cygwin is called `gcc-g++`.
I have added the information to the package prereq table, and the paragraph you commented on now references that section.
DONE.